plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -470,7 +470,7 @@ def main():
         password=dict(type='str', no_log=True),
         email=dict(type='str'),
         sshkey_name=dict(type='str'),
-        sshkey_file=dict(type='str'),
+        sshkey_file=dict(type='str', no_log=False),
```

Comment:
This seems to be a private key: https://github.com/ansible-collections/community.general/blob/3053a85e567fd5a70e6801b464f04f9b5b266672/plugins/modules/source_control/gitlab/gitlab_user.py#L62

Reply:
From the example (and from how GitLab/GitHub work), it is a public key: https://github.com/ansible-collections/community.general/blob/3053a85e567fd5a70e6801b464f04f9b5b266672/plugins/modules/source_control/gitlab/gitlab_user.py#L126

Reply:
Sorry, you're right!

Reply:
No problem, thanks a lot for checking :) I tried to check all of them, but I don't know most of the modules and am thus very happy if someone else takes a look as well (and in particular when it turns out I didn't screw it up ;) )!